code/modules/awaymissions/mission_code/ruins/telecomns.dm

```diff
@@ -0,0 +1,495 @@
+// stuff for the telecomms sat (telecomms_returns.dmm)
+
+/obj/effect/abstract/bot_trap
+	name = "evil bot trap to make explorers hate you"
+
+/obj/effect/abstract/bot_trap/Crossed(atom/movable/AM, oldloc)
+	. = ..()
+	if(isrobot(AM) || ishuman(AM))
+		var/turf/T = get_turf(src)
+		for(var/mob/living/simple_animal/bot/B in urange(22, T))
```

Comment:
Would it be possible to use a GLOB for this? This might be *incredibly* expensive, especially if it's crossed multiple times in a row.

Reply:
It qdels after being trigged, but can do